commerce_coordinator/settings/base.py

```diff
@@ -345,7 +345,7 @@ def root(*path_fragments):
 FULFILLMENT_TIMEOUT = 7
 
 # API URLs
-ECOMMERCE_URL = 'https://ecommerce_url/'
+ECOMMERCE_URL = 'http://localhost:18130/'
```

Comment:
To match it with the value set in edx-internal, the URL base should not have a trailing slash.

```suggestion
ECOMMERCE_URL = 'http://localhost:18130'
```